sympy/stats/drv_types.py

```diff
@@ -434,6 +434,11 @@ def _characteristic_function(self, t):
     def _moment_generating_function(self, t):
         return exp(self.lamda * (exp(t) - 1))
 
+    def _cdf(self, t):
+        s = Symbol("s")
+        t_summation = summation((1/factorial(s)) * ((self.lamda) ** s), (s, 0, floor(t)))
```

Comment:
IMO, it should return `Sum` object and not `summation`. What do you say @czgdp1807?

Reply:
@Smit-create can you elaborate like why it should return `Sum` object? 
As I am unable to get your question...

Reply:
Actually, I'd expect the actual expanded formula than using anything like Sum or summation. The first one could be a lazy evaluation, but it is not appropriate here because this is supposed to do the actual computation.
And the second one still suffers from inefficiency because it evaluates summation through generalized symbolic summation algorithms.
In here, I see there is a closed form CDF with gamma and rounding functions
https://en.wikipedia.org/wiki/Poisson_distribution
if this can be mathematically verified to be same with what sympy had used before, it can be used instead.

Reply:
@sylee957 Did you expect something like:-
```
uppergamma(t+1, lamda).evalf()/factorial(t)
```
if that is the case I tried it and yup its giving exact results as the results were given before.
Consider this case:-
```
>>> l = 3
>>> p = PoissonDistribution(l)
>>> p.cdf(10).evalf()
0.999707663049353

>>> uppergamma(10 + 1, l).evalf()/factorial(10)
0.999707663049353
```
So can I implement this?